### DISCUSSION TITLE
A problem when using reverse proxy

### DISCUSSION THREAD

**Discussion by hhhhhliu** [ckulka/baikal-docker#59] [Q&A] [answered]
When I use reverse proxy to access the web page through https://my.domain:port/dav/, the "/dav/" makes the resource unable to load, i.e., it can only be accessed through "/". Which configuration file should I modify to solve this problem?

Another small question: Are there any measures such as login frequency restrictions to prevent brute-force cracking of admin's password?

**Comment by ckulka** [comment, marked as answer]
Hi @hhhhhliu ,

while I haven't tested it, I guess you have two options (assuming you are on version 0.7.1):

1. Use the `base_uri` configuration option in `config/baikal.yaml`, e.g.
   ```yaml
   system:
     # ...
     base_uri: 'dav/'
     # ...
   ```
   Reference: https://github.com/sabre-io/Baikal/issues/943
2. Strip the path prefix with Nginx or Apache httpd

I'd go with option 1, but I don't know your setup and if that would really work better for you.

For the small question, I would try Nginx's rate limiting feature: <https://www.nginx.com/blog/rate-limiting-nginx/>. Apache httpd has a similar feature called `mod_ratelimit`: <https://httpd.apache.org/docs/2.4/mod/mod_ratelimit.html>. Again, not sure what setup you have, but those two options seem promising and the simplest way to implement it.

Also (and I figure this is obvious) this is no real protection. Unless you spend a lot more time setting up alerting to detect attacks, additional hardening, ..., or an actual WAF to protect your Baikal instance, this is (imho) virtually a moot point. And that's on top of the usual PHP security bashing :)

Hope this helps
